api/proto/v1/network.proto

```diff
@@ -52,28 +52,6 @@ message InternalNetworkInfo {
     repeated InternalSecurityGroupInfo security_groups = 10;
 }
 
-message InternalServiceInfo {
```

Comment:
Hi @cj-chung, the InternalServiceInfo and InternalComputeInfo may still be needed for the Merak Network component

Reply:
@cj-chung I've move them to `common.proto`, since compute also need the message definition.

Reply:
sounds good, thanks!